src/formatDate.js

```diff
@@ -50,7 +50,37 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDate = date.split(fromFormat[3]);
+  const newDate = [];
+
+  const oldYear = fromFormat.includes('YYYY')
+    ? fromFormat.indexOf('YYYY') : fromFormat.indexOf('YY');
+  const oldMonth = fromFormat.indexOf('MM');
+  const oldDay = fromFormat.indexOf('DD');
+
+  const newYear = toFormat.includes('YYYY')
+    ? toFormat.indexOf('YYYY') : toFormat.indexOf('YY');
+  const newMonth = toFormat.indexOf('MM');
+  const newDay = toFormat.indexOf('DD');
```

Comment:
You kind of repeat your code twice. Better create object for this purpose and put data here, using loop.